File: lab 2.cpp
#include<iostream>
using namespace std;

//this is a class
 class Numsum{
     public:
    int num1,num2,num3;

//this is a constructor
    Numsum( int n1,int n2,int n3){
         num1=n1;
         num2=n2;
         num3=n3;
    }

//this is actually a method
    int  calculateSum(){
         return num1+num2+num3;
    }
 };

//this is a main function
    int main(){
        int num1,num2,num3;
        
        cout<< "Enter the value of num1 :" ;
        cin>> num1;
        cout<< "Enter the vakue of num2 :";
        cin>> num2;
        cout <<"Enter the value of num3:";
        cin>> num3;

     // this is a object
        Numsum sum(num1,num2,num3);
        cout<<"Sum of the three value"<<sum.calculateSum() << endl;
        return 0;
    }

File: lab 2.java
2. Write a program to print the sum of the three numbers entered by user by creating a class and
method.( By C++ or Java )



import java.util.Scanner;

// CLASS
public class NumSum{
    int num1,num2,num3;

  //
    NumSum(int n1,int n2,int n3){
        this.num1=n1;
        this.num2=n2;
        this.num3=n3;
    }
     public int calculateSum (){
        return num1+num2+num3;
    }

    public static void main(String[] args){
        int num1,num2,num3;
        Scanner scanner=new Scanner(System.in);
        System.out.println("Enter the value of num1");
        num1=scanner.nextInt();
        System.out.println("Enter the value of num2");
        num2=scanner.nextInt();
        System.out.println("Enter the value of num3");
        num3=scanner.nextInt();
    NumSum nsum=new NumSum(num1,num2,num3);
    System.out.println("Sum of the three number" + nsum.calculateSum());
    scanner.close ();
}
}
